JitsiConferenceEventManager.js

```diff
@@ -597,9 +578,9 @@ JitsiConferenceEventManager.prototype.setupStatisticsListeners = function () {
     });
 
     conference.statistics.addByteSentStatsListener(function (stats) {
-        conference.getLocalTracks().forEach(function (track) {
-            var ssrc = track.getSSRC();
-            if(!track.isAudioTrack() || !ssrc || !stats.hasOwnProperty(ssrc))
+        conference.getLocalTracks(MediaType.AUDIO).forEach(function (track) {
```

Comment:
the previous code seemed to return early if the track wasn't an audio track, AFAIS.